packages/xod-arduino-deploy/src/packageManager.js

```diff
@@ -8,6 +10,16 @@ import { tapP } from 'xod-func-tools';
 
 import * as Utils from './utils';
 
+// =============================================================================
+//
+// HTTP/HTTPS agents
+//
+// =============================================================================
+const httpsAgent = new https.Agent({
+  rejectUnauthorized: false,
```

Comment:
That’s not a solution, but rather a security hole.

The problem should be solved by fixing the CA/valid cert lists.